towhee/compiler/operator_metaclass.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def op_action(func):
```

Comment:
why op_action  both in perator_metaclass.py  and operator_decorator.py

Reply:
I forgot to remove op_action from operator_metaclass.py. 
will fix it in the next pr @junjiejiangjjj

Reply:
fixed in #7 @junjiejiangjjj